<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_Asserv</class>
 <widget class="QMainWindow" name="ihm_Asserv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_10">
    <item row="0" column="0" rowspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-199</y>
         <width>232</width>
         <height>482</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>COMMANDE MANUELLE</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Gauche</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="CdeManuelle_Gauche">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="CdeManuelle_SendLock">
             <property name="text">
              <string>Send/Lock</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Droite</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="CdeManuelle_Droit"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Stop All</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>COMMANDE XY</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="CdeXY_X"/>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="QPushButton" name="CdeXY_SendLock">
             <property name="text">
              <string>Send/Lock</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="CdeXY_Y"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>COMMANDE XY-TETA</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="CdeXYTeta_X"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="CdeXYTeta_Y"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="CdeXYTeta_SendLock">
             <property name="text">
              <string>Send/Lock</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>TETA</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="CdeXYTeta_Teta"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>COMMANDE DISTANCE ANGLE</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>DISTANCE</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="CdeDistAngle_Distance"/>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="QPushButton" name="CdeDistAngle_SendLock">
             <property name="text">
              <string>Send/Lock</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>ANGLE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="CdeDistAngle_Angle"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>COMMANDE REGUL VITESSE ROUES</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Roue Gauche</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="CdeRegulVitesse_Gauche"/>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="QPushButton" name="CdeRegulVitesse_SendLock">
             <property name="text">
              <string>Send/Lock</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Roue Droite</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="CdeRegulVitesse_Droite"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>VITESSE MOUVEMENT</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>vitesse avance max</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_13"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>vitesse rotation max</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_14"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Send/Lock</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>indice sportivite accel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_16"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>indice sportivite decel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_17"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>REINIT XY-TETA</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_10"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_11"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Send/Lock</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>TETA</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_12"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="title">
       <string>ETAT ASSERVISSEMENT</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Cde mot G</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="Etat_CdeMotGauche">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="5">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>X robot</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSpinBox" name="PosRobot_X">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Cde mot D</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="Etat_CdeMotDroit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Y robot</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSpinBox" name="PosRobbot_Y">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Convergence</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="EtatConvergence"/>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>TETA robot</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QSpinBox" name="PosRobot_Teta">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Mode Asserv</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>cpt diag blocage</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBox_21">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
